aries-site/src/pages/components/card/index.mdx

```diff
@@ -133,34 +133,7 @@ When cards are presenting as a group, they should function as a list.
 ## Misuses
 
 - When data attributes across items needs to be analyzed, sorted, compared, and filtered, a card may not be the best tool. Instead, [DataTable](/components/datatable) may be more appropriate tool to meet these needs.
-- Cards should not be used as a control to filter data on the same screen. A button is the appropriate control to apply a filter. Alternatively, cards may be used to present summarized information and allow a user to navigate to a pre-filtered view of that information.
-
-<BestPracticeGroup>
-  <Example
-    height={{ min: 'small' }}
-    bestPractice={{
-      type: 'do',
-      message:
-        'Style status filter buttons as default buttons when the buttons are placed on the same page as the data they will be filtering.',
-    }}
-    scale={0.4}
-    width="100%"
-  >
-    <CardFilteringBestPractice />
-  </Example>
-  <Example
-    height={{ min: 'small' }}
-    bestPractice={{
-      type: 'dont',
-      message: `Don’t use a card to wrap a set of status filter buttons if clicking on one of those buttons will filter data that exists on the same page as that card. Cards should act as independent regions that do not affect surrounding content. However, placing these buttons in a card would be appropriate on a dashboard, where clicking on one of the status buttons would navigate to a different page that is filtered by the selected status.`,
-    }}
-    scale={0.4}
-    width="100%"
-  >
-    <CardFilteringBestPractice bestPractice={false} />
-  </Example>
-</BestPracticeGroup>
-
+- Cards should not be used as controls to filter data directly on the same screen. Instead, the [Selector](templates/selector) component is the appropriate tool for filtering data sets.
```

Comment:
```suggestion
- Cards should not be used as controls to filter data directly on the same screen. Instead, the [selector](templates/selector) component is the appropriate tool for filtering data sets.
```

think we dont capitalize component names @taysea can you confirm

Reply:
![Screenshot 2024-09-26 at 11 45 42 AM](https://github.com/user-attachments/assets/a97c4848-14dd-46bd-adc1-41968be448ea)
Shall we change DataTable too?

Reply:
I think in this scenario since we are referring directly to the component, capitalized is fine!